poutyne/framework/callbacks/callbacks.py

```diff
@@ -141,17 +135,14 @@ def on_epoch_end(self, epoch, logs):
                  * 'epoch': The epoch number.
                  * 'loss': The average loss of the batches.
                  * 'time': The computation time of the epoch.
-                 * Other metrics: One key for each type of metrics. The metrics
-                   are also averaged.
-                 * val_loss': The average loss of the batches on the validation
-                   set.
-                 * Other metrics: One key for each type of metrics on the
-                   validation set. The metrics are also averaged.
+                 * Other metrics: One key for each type of metrics. The metrics are also averaged.
+                 * val_loss': The average loss of the batches on the validation set.
+                 * Other metrics: One key for each type of metrics on the validation set. The metrics are also averaged.
 
         Example::
 
-            logs = {'epoch': 6, 'time': 3.141519837, 'loss': 4.34462, 'accuracy': 0.766,
-                    'val_loss': 5.2352, 'val_accuracy': 0.682}
+            logs = {'epoch': 6, 'time': 3.141519837, 'loss': 4.34462,
+                    'accuracy': 0.766, 'val_loss': 5.2352, 'val_accuracy': 0.682}
```

Comment:
why is accuracy on the next line here?

Reply:
I moved it because it made the two lines more equal, but I guess having the val stats on their own line makes more sense.

Reply:
Fixed by d3b8dbbe0d45e523105f6a3c56b044bdaafcfd47.